cfgov/unprocessed/js/modules/ExternalSite.js

```diff
@@ -0,0 +1,82 @@
+/* ==========================================================================
+   External Site Initialization
+   Used on at least `/external-site/`.
+   ========================================================================== */
+
+'use strict';
+
+var atomicCheckers = require( '../modules/util/atomic-checkers' );
+
+/**
+ * ExternalSite
+ * @class
+ *
+ * @classdesc Initializes a new ExternalSite instance.
+ *
+ * @param {HTMLElement} element DOM Element.
+ */
+function ExternalSite( element ) {
+
+  var BASE_CLASS = 'external-site_container';
+  var TOTAL_DURATION = 5;
+  var INTERVAL = 1000;
+
+  var _dom =
+    atomicCheckers.validateDomElement( element, BASE_CLASS, 'ExternalSite' );
```

Comment:
Does the `validateDomElement` method just return a dom element if it exists and trigger an error if not?